tests/kernel_bundle/use_kernel_bundle_call_set_spec_const_before_use_kb_no_second_queue.cpp

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This test calls sycl::handler::use_kernel_bundle after calling
+//  sycl::handler::set_specialization_constant without secondary queue.
+//
+//  The test verifies that the exception with sycl::errc::invalid was thrown.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "use_kernel_bundle.h"
+
+#define TEST_NAME \
+  use_kernel_bundle_call_set_spec_const_before_use_kb_no_second_queue
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace sycl_cts::tests::use_kernel_bundle;
+
+constexpr sycl::specialization_id<int> spec_const;
+
+/** @brief Call sycl::handler::set_specialization_constant after
+ *         sycl::handler::use_kernel_bundle with secondary queue
+ *  @param log sycl_cts::util::logger class object
+ *  @param ctx Context that will be used for constructing sycl::queue
+ *  @param kernel_bundle empty, non-empty or kernel_bundle with built-in kernels
+ */
+void run_verification(
+    sycl_cts::util::logger &log, const sycl::context &ctx,
+    sycl::kernel_bundle<sycl::bundle_state::executable> &kernel_bundle) {
+  sycl::queue queue(ctx, ctx.get_devices()[0]);
+  bool ex_was_thrown = false;
+  try {
+    queue.submit([&](sycl::handler &cgh) {
+      cgh.set_specialization_constant<spec_const>(0);
+      cgh.use_kernel_bundle(kernel_bundle);
+      cgh.single_task<kernel_for_kernel_bundle>([=]() {});
```

Comment:
```suggestion
      cgh.single_task<kernel_for_kernel_bundle>([=] {});
```

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/225/commits/e157b8bd72174f5b18b4e387b9dfb4e353573bd7#diff-d25bca94ae123b3ee37637a07382213babede7e443b70583f54eedd749ceec2dL39-L40) and in other cases